libsrc/apple2/ser/a2gs.ssc.inc

```diff
@@ -0,0 +1,97 @@
+
+; ------------------------------------------------------------------------
+; Write registers, read registers, and values that interest us
+
+WR_INIT_CTRL           := 0
+RR_INIT_STATUS         := 0
+INIT_CTRL_CLEAR_EIRQ   := %00010000
+INIT_CTRL_CLEAR_ERR    := %00110000
+INIT_STATUS_READY      := %00000100
+INIT_STATUS_RTS        := %00100000
+
+WR_TX_RX_MODE_CTRL     := 1
+TX_RX_MODE_OFF         := %00000000
+TX_RX_MODE_RXIRQ       := %00010001
+
+WR_RX_CTRL             := 3            ; (Ref page 5-7)
+RR_RX_STATUS           := 9            ; Corresponding status register
+RX_CTRL_ON             := %00000001    ; ORed, Rx enabled
+RX_CTRL_OFF            := %11111110    ; ANDed,Rx disabled
+
+WR_TX_RX_CTRL          := 4
+RR_TX_RX_STATUS        := 4
+TX_RX_CLOCK_MUL        := %01000000    ; Clock x16 (Ref page 5-8)
+
+WR_TX_CTRL             := 5            ; (Ref page 5-9)
+RR_TX_STATUS           := 5            ; Corresponding status register
+TX_CTRL_ON             := %00001000    ; ORed, Tx enabled
+TX_CTRL_OFF            := %11110111    ; ANDed,Tx disabled
+TX_DTR_ON              := %01111111    ; ANDed,DTR ON (high)
+TX_DTR_OFF             := %10000000    ; ORed, DTR OFF
+TX_RTS_ON              := %00000010    ; ORed, RTS ON (low)
+TX_RTS_OFF             := %11111101    ; ANDed, RTS OFF
+
+WR_MASTER_IRQ_RST      := 9            ; (Ref page 5-14)
+MASTER_IRQ_SHUTDOWN    := %00000010    ; STA'd
+MASTER_IRQ_MIE_RST     := %00001010    ; STA'd
+MASTER_IRQ_SET         := %00011001    ; STA'd
+
+WR_CLOCK_CTRL          := 11           ; (Ref page 5-17)
+CLOCK_CTRL_CH_A        := %11010000
+CLOCK_CTRL_CH_B        := %01010000
+
+WR_BAUDL_CTRL          := 12           ; (Ref page 5-18)
+WR_BAUDH_CTRL          := 13           ; (Ref page 5-19)
+
+WR_MISC_CTRL           := 14           ; (Ref page 5-19)
+MISC_CTRL_RATE_GEN_ON  := %00000001    ; ORed
+MISC_CTRL_RATE_GEN_OFF := %11111110    ; ANDed
+
+WR_IRQ_CTRL            := 15           ; (Ref page 5-20)
+IRQ_CLEANUP_EIRQ       := %00001000
+
+RR_SPEC_COND_STATUS    := 1            ; (Ref page 5-23)
+SPEC_COND_FRAMING_ERR  := %01000000
+SPEC_COND_OVERRUN_ERR  := %00100000
+
+RR_IRQ_STATUS          := 2            ; (Ref page 5-24)
+IRQ_MASQ               := %01110000    ; ANDed
+IRQ_RX                 := %00100000
+IRQ_SPECIAL            := %01100000
+
+RR_INTR_PENDING_STATUS := 3            ; (Ref page 5-25)
+INTR_PENDING_RX_EXT_A  := %00101000    ; ANDed (RX or special IRQ)
+INTR_PENDING_RX_EXT_B  := %00000101    ; ANDed (RX or special IRQ)
+INTR_IS_RX             := %00100100    ; ANDed (RX IRQ, channel A or B)
+
+; We're supposed to get SerFlag's address using GetAddr on ROMs 1 and 3.
+; (https://archive.org/details/IIgs_2523018_SCC_Access, page 9)
+; But, it's the same value as on ROM0. As we don't expect a ROM 4 anytime
+; soon with a different value, let's keep it simple.
+
+SER_FLAG               := $E10104
+SER_FLAG_CH_A          := %00111000
+SER_FLAG_CH_B          := %00000111
```

Comment:
Please use the label syntax `:=` only for addresses (and the constant syntax `=` for all other things. See https://cc65.github.io/doc/ca65.html#ss6.1

I know that the code base doesn't consistently adhere to that, but here's an example where I did it right: https://github.com/cc65/cc65/blob/master/libsrc/apple2/mou/a2.stdmou.s#L15-L29